packages/eslint-config/eslint-config.cjs

```diff
@@ -77,8 +77,8 @@ module.exports = {
           '**/*.config.*.js',
           // leading wildcard to work in CLI (package path) and IDE (repo path)
           '**/test/**',
-          '**/demo*/**/*.js',
-          '**/scripts/**/*.js',
```

Comment:
Was this change intentional? There are other regexs above that are of old format.

Reply:
yes, to match `.mjs`: [ci(lint): glob mjs for devDependencies](https://github.com/Agoric/agoric-sdk/pull/7260/commits/b2a2d187698c809d457efa64994fa09e36a293b0)

The regex above don't need it though it wouldn't hurt to update

Reply:
Any concern that `*js` can have false hits like `Swarājs` ?